glean-core/android/metrics.yaml

```diff
@@ -86,3 +86,21 @@ glean.validation:
     expires: 2019-12-20
     send_in_pings:
       - baseline
+
+  app_forceclosed_count:
```

Comment:
Is this missing a `send_in_pings` statement?

Reply:
Yes, that's intentional. Glean sends this automatically in the `metrics` ping.

Reply:
And here I am, always thinking that `send_in_pings: default` would be required.